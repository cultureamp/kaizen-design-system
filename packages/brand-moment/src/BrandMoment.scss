@import "~@kaizen/design-tokens/sass/color-vars";
@import "~@kaizen/design-tokens/sass/spacing-vars";
@import "~@kaizen/design-tokens/sass/layout";
@import "~@kaizen/design-tokens/sass/layout-vars";

$breakpoint-xl: 1366px; // TODO: create an xl breakpoint in design-tokens and replace this out

// --------------------------------
// Body
// --------------------------------

.body {
  width: 100%;
  display: flex;
  flex-direction: column;
  align-items: center;

  &.informative {
    background: $kz-var-color-cluny-100;
  }

  &.positive {
    background: $kz-var-color-seedling-100;
  }

  &.negative {
    background: $kz-var-color-coral-100;
  }
}

@media (min-width: $kz-layout-breakpoints-large) {
  .body {
    min-height: 100vh;
  }
}

// --------------------------------
// Reusable container
// --------------------------------

.container {
  max-width: $kz-var-layout-content-max-width;
  margin: 0 auto;
  padding: 0 $kz-var-spacing-md;
}

@media (min-width: $kz-layout-breakpoints-large) {
  .container {
    padding: 0 $kz-var-spacing-xxl;
  }
}

@media (min-width: $breakpoint-xl) {
  .container {
    padding: 0 $kz-var-spacing-xxxxl;
  }
}

// --------------------------------
// Header
// --------------------------------

.header {
  width: 100%;
  margin-bottom: $kz-var-spacing-md;
}

// --------------------------------
// Main
// --------------------------------

.main {
  flex: 1 0 auto;
  display: flex;
  align-items: center;
  margin-bottom: $kz-var-spacing-xxl;
}

.mainInner {
  display: grid;
  grid-template-columns: 1fr;
  gap: $kz-var-spacing-xl $kz-var-spacing-lg;
  align-items: center;
}

@media (min-width: $kz-layout-breakpoints-medium) {
  .mainInner {
    row-gap: $kz-var-spacing-xxl;
  }
}

@media (min-width: $kz-layout-breakpoints-large) {
  .mainInner {
    grid-template-columns: 1fr 1fr;
  }
}

@media (min-width: $breakpoint-xl) {
  .mainInner {
    column-gap: $kz-var-spacing-xxxxl;
  }
}

.left {
  display: flex;
  justify-content: center;
}

.leftInner {
  width: 100%;
  max-width: 525px;
}

.right {
  flex-direction: column;
  display: flex;
  justify-content: center;
  align-items: center;
}

.rightInner {
  max-width: 543px;
}

.actions {
  display: flex;
}

@media (min-width: $kz-layout-breakpoints-large) {
  .left {
    justify-content: flex-end;
  }

  .right {
    align-items: flex-start;
  }
}

// --------------------------------
// Footer
// --------------------------------

.footer {
  flex: 0 0 auto;
}

.footerInner {
  display: flex;
  flex-direction: column;
  border-top: 1px solid $kz-var-color-wisteria-800;
  border-left: 1px solid $kz-var-color-wisteria-800;
  padding: $kz-var-spacing-sm $kz-var-spacing-md;
  margin: 0 0 $kz-var-spacing-lg;
}

.poweredByContainer {
  flex: 0 0 auto;
  order: 2;
  display: flex;
}

.footerTextContainer {
  flex: 1 1 auto;
  order: 1;
  margin-bottom: $kz-var-spacing-md;
}

.poweredByLogo {
  margin-left: $kz-var-spacing-xs;

  > img {
    max-width: 133px;
  }
}

@media (min-width: $kz-layout-breakpoints-large) {
  .footerInner {
    flex-direction: row;
    align-items: center;
  }

  .footerTextContainer {
    order: initial;
    margin-bottom: 0;
    margin-left: $kz-var-spacing-md;
  }

  .poweredByContainer {
    order: initial;
  }
}
