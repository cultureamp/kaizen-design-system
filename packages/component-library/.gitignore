components/**/*.js
components/**/*.js.map
components/**/*.d.ts
draft/**/*.js
draft/**/*.js.map
draft/**/*.d.ts
util/**/*.js
util/**/*.js.map
util/**/*.d.ts
index.js
index.js.map
index.d.ts
