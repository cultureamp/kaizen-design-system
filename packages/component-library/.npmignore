*.ts
!*.d.ts
/draft-templates/
/mocks/
/stories/
.gitignore
.npmignore
tsconfig.dist.json
__snapshots__/
*.spec.*