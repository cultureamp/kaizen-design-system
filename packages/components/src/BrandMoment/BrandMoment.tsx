import React, { ReactNode, ReactElement, HTMLAttributes } from 'react'
import classnames from 'classnames'
import { Heading } from '~components/Heading'
import { SceneProps } from '~components/Illustration'
import { Text } from '~components/Text'
import { Button, ButtonProps } from '~components/__actions__/v2'
import { OverrideClassName } from '~components/types/OverrideClassName'
import { assetUrl } from '~components/utils/hostedAssets'
import { useMediaQueries } from '~components/utils/useMediaQueries'
import styles from './BrandMoment.module.scss'

type DeprecatedMoodProps = {
  /** @deprecated Use `variant` instead */
  mood: 'informative' | 'positive' | 'negative'
  variant?: never
}

type VariantProps = {
  /**
   * If you are transitioning from `mood`:
   * - `informative` remains as `informative`
   * - `positive` should be `success`
   * - `negative` should be `warning`
   */
  variant: 'informative' | 'success' | 'warning'
  /** @deprecated Use `variant` instead */
  mood?: never
}

export type BrandMomentProps = {
  illustration: ReactElement<SceneProps>
  header: ReactNode
  body?: ReactNode
  primaryAction?: ButtonProps
  secondaryAction?: ButtonProps
  text: {
    title: ReactNode
    subtitle?: ReactNode
    body?: ReactNode
    footer?: ReactNode
  }
} & OverrideClassName<HTMLAttributes<HTMLDivElement>> &
  (DeprecatedMoodProps | VariantProps)

/**
 * {@link https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3082061589/Brand+Moment Guidance} |
 * {@link https://cultureamp.design/?path=/docs/components-brand-moment--docs Storybook}
 */
export const BrandMoment = ({
  mood,
  variant,
  illustration,
  header,
  body,
  primaryAction,
  secondaryAction,
  text,
  classNameOverride,
  ...restProps
}: BrandMomentProps): JSX.Element => {
  const { queries } = useMediaQueries()

  return (
    <div
      className={classnames(
        styles.body,
        variant ? styles[variant] : styles[mood],
        classNameOverride,
      )}
      {...restProps}
    >
      <header className={styles.header}>{header}</header>
      <main className={styles.main}>
        <div className={styles.container}>
          <div className={styles.mainInner}>
            <div className={styles.left}>
              <div className={styles.leftInner}>{illustration}</div>
            </div>
            <div className={styles.right}>
              <div className={styles.rightInner}>
                {text.subtitle && (
                  <Heading
                    variant="heading-3"
                    tag="h1"
                    classNameOverride={styles.subtitle}
                  >
                    {text.subtitle}
                  </Heading>
                )}
                <Heading
                  variant="display-0"
                  tag={text.subtitle ? 'h2' : 'h1'}
                  classNameOverride={styles.title}
                >
                  {text.title}
                </Heading>
                {text.body && (
                  <Text
                    variant="intro-lede"
                    classNameOverride={styles.textBody}
                  >
                    {text.body}
                  </Text>
                )}
                {body && <div className={styles.textBody}>{body}</div>}
                <div className={styles.actions}>
                  {primaryAction && (
                    <Button
                      primary
                      fullWidth={queries.isSmall}
                      {...primaryAction}
                    />
                  )}
                  {secondaryAction && (
                    <div className={styles.secondaryAction}>
                      <Button
                        secondary
                        fullWidth={queries.isSmall}
                        {...secondaryAction}
                      />
                    </div>
                  )}
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
      {text.footer && (
        <footer className={styles.footer}>
          <div className={styles.container}>
            <div className={styles.footerInner}>
              <div className={styles.poweredByContainer}>
                <Text variant="extra-small" color="dark-reduced-opacity">
                  Powered by
                </Text>
                <a
                  href="https://www.cultureamp.com"
                  className={styles.poweredByLogo}
                >
                  <img
                    src={assetUrl('brand/logo-horizontal-default.svg')}
                    alt="Culture Amp"
                  />
                </a>
              </div>
              <div className={styles.footerTextContainer}>
                <Text variant="extra-small">{text.footer}</Text>
              </div>
            </div>
          </div>
        </footer>
      )}
    </div>
  )
}

BrandMoment.displayName = 'BrandMoment'
