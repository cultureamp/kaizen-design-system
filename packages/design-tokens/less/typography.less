/** THIS IS AN AUTOGENERATED FILE **/

@kz-typography-display-0-font-family: "Greycliff CF";
@kz-typography-display-0-font-weight: 700;
@kz-typography-display-0-font-size: 4.5rem;
@kz-typography-display-0-line-height: 4.5rem;
@kz-typography-display-0-letter-spacing: -0.03em;
@kz-typography-heading-1-font-family: "Greycliff CF";
@kz-typography-heading-1-font-weight: 700;
@kz-typography-heading-1-font-size: 2.25rem;
@kz-typography-heading-1-line-height: 2.25rem;
@kz-typography-heading-1-letter-spacing: -0.03em;
@kz-typography-heading-2-font-family: "Greycliff CF";
@kz-typography-heading-2-font-weight: 700;
@kz-typography-heading-2-font-size: 1.875rem;
@kz-typography-heading-2-line-height: 2.25rem;
@kz-typography-heading-2-letter-spacing: -0.03em;
@kz-typography-heading-3-font-family: "Greycliff CF";
@kz-typography-heading-3-font-weight: 700;
@kz-typography-heading-3-font-size: 1.375rem;
@kz-typography-heading-3-line-height: 1.5rem;
@kz-typography-heading-3-letter-spacing: -0.03em;
@kz-typography-heading-4-font-family: "Inter";
@kz-typography-heading-4-font-weight: 600;
@kz-typography-heading-4-font-size: 1.125rem;
@kz-typography-heading-4-line-height: 1.5rem;
@kz-typography-heading-4-letter-spacing: normal;
@kz-typography-heading-5-font-family: "Inter";
@kz-typography-heading-5-font-weight: 600;
@kz-typography-heading-5-font-size: 1rem;
@kz-typography-heading-5-line-height: 1.5rem;
@kz-typography-heading-5-letter-spacing: normal;
@kz-typography-heading-6-font-family: "Inter";
@kz-typography-heading-6-font-weight: 700;
@kz-typography-heading-6-font-size: 0.875rem;
@kz-typography-heading-6-line-height: 1.5rem;
@kz-typography-heading-6-letter-spacing: normal;
@kz-typography-paragraph-intro-lede-font-family: "Inter";
@kz-typography-paragraph-intro-lede-font-weight: 400;
@kz-typography-paragraph-intro-lede-font-size: 1.25rem;
@kz-typography-paragraph-intro-lede-line-height: 1.875rem;
@kz-typography-paragraph-intro-lede-letter-spacing: -0.5px;
@kz-typography-paragraph-body-font-family: "Inter";
@kz-typography-paragraph-body-font-weight: 400;
@kz-typography-paragraph-body-font-size: 1rem;
@kz-typography-paragraph-body-line-height: 1.5rem;
@kz-typography-paragraph-body-letter-spacing: normal;
@kz-typography-paragraph-small-font-family: "Inter";
@kz-typography-paragraph-small-font-weight: 400;
@kz-typography-paragraph-small-font-size: 0.875rem;
@kz-typography-paragraph-small-line-height: 1.125rem;
@kz-typography-paragraph-small-letter-spacing: normal;
@kz-typography-paragraph-extra-small-font-family: "Inter";
@kz-typography-paragraph-extra-small-font-weight: 400;
@kz-typography-paragraph-extra-small-font-size: 0.75rem;
@kz-typography-paragraph-extra-small-line-height: 1.125rem;
@kz-typography-paragraph-extra-small-letter-spacing: normal;
