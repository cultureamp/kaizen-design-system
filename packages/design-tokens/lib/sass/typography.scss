/** THIS IS AN AUTOGENERATED FILE **/

$font-heading-family: 'Greycliff CF';
$font-heading-size--0: 4.5rem;
$font-heading-size--1: 2.25rem;
$font-heading-size--2: 1.875rem;
$font-heading-size--3: 1.375rem;
$font-heading-size--4: 1.125rem;
$font-heading-size--5: 1rem;
$font-heading-size--6: 0.875rem;
$font-paragraph-family: 'Inter';
$font-paragraph-size--4: 1.25rem;
$font-paragraph-size--5: 1rem;
$font-paragraph-size--6: 0.875rem;
$font-paragraph-size--7: 0.75rem;
$font-preformatted-family: 'IBM Plex Mono';
$font-preformatted-size--5: 1;
$DEPRECATED-font-default-family: 'Open Sans';
$DEPRECATED-font-sans-serif-family: 'Ideal Sans A';
