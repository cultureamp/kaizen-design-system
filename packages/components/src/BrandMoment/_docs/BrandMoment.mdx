import { Canvas, Meta } from "@storybook/blocks"
import { InlineNotification } from "~components/Notification"
import { ResourceLinks, KAIOInstallation } from "~storybook/components"
import * as BrandMoment from "./BrandMoment.stories"

<Meta of={BrandMoment} />

# BrandMoment

<ResourceLinks
  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/BrandMoment"
  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%F0%9F%96%BC%EF%B8%8F-Component-Gallery?type=design&node-id=9-132873&mode=design&t=ITSkv2Fu67xK6kBC-0"
  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3082061589/Brand+Moment"

/>

<KAIOInstallation exportNames="BrandMoment" />

## Overview

A brand moment is a page that showcases the brand's vibrant animations and illustration. The page provides an engaging experience for users when they reach these junction, helping to create a strong connection between the brand and product.

<InlineNotification persistent type="informative">Code snippets on this page have been disabled due to performance issues, see [Source Code](https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/BrandMoment) for code examples.</InlineNotification>

## Variants

### Informative Intro
<Canvas of={BrandMoment.InformativeIntro} />

### Positive Outro
<Canvas of={BrandMoment.PositiveOutro} />

### Positive Outro Customer Focused
<Canvas of={BrandMoment.PositiveOutroCustomerFocused} />

### Error
<Canvas of={BrandMoment.Error} />
