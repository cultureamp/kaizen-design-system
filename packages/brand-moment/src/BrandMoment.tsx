import React, { ReactNode, ReactElement, HTMLAttributes } from "react"
import classnames from "classnames"
import { Button, ButtonProps } from "@kaizen/button"
import { OverrideClassName } from "@kaizen/component-base"
import { Box } from "@kaizen/component-library"
import { SceneProps } from "@kaizen/draft-illustration"
import { assetUrl } from "@kaizen/hosted-assets"
import { useMediaQueries } from "@kaizen/responsive"
import { Heading, Paragraph } from "@kaizen/typography"
import styles from "./BrandMoment.module.scss"

export interface BrandMomentProps
  extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
  mood: "informative" | "positive" | "negative"
  illustration: ReactElement<SceneProps>
  header: ReactNode
  body?: ReactNode
  primaryAction?: ButtonProps
  secondaryAction?: ButtonProps
  text: {
    title: ReactNode
    subtitle?: ReactNode
    body?: ReactNode
    footer?: ReactNode
  }
}

export const BrandMoment = ({
  mood,
  illustration,
  header,
  body,
  primaryAction,
  secondaryAction,
  text,
  classNameOverride,
  ...restProps
}: BrandMomentProps): JSX.Element => {
  const { queries } = useMediaQueries()

  return (
    <div
      className={classnames(styles.body, styles[mood], classNameOverride)}
      {...restProps}
    >
      <header className={styles.header}>{header}</header>
      <main className={styles.main}>
        <div className={styles.container}>
          <div className={styles.mainInner}>
            <div className={styles.left}>
              <div className={styles.leftInner}>{illustration}</div>
            </div>
            <div className={styles.right}>
              <div className={styles.rightInner}>
                {text.subtitle && (
                  <Box mb={0.5}>
                    <Heading variant="heading-3" tag="h1">
                      {text.subtitle}
                    </Heading>
                  </Box>
                )}
                <Box mb={1.5}>
                  <Heading
                    variant="display-0"
                    tag={text.subtitle ? "h2" : "h1"}
                  >
                    {text.title}
                  </Heading>
                </Box>
                {text.body && (
                  <Box mb={1.5}>
                    <Paragraph variant="intro-lede">{text.body}</Paragraph>
                  </Box>
                )}
                {body && <Box mb={1.5}>{body}</Box>}
                <div className={styles.actions}>
                  {primaryAction && (
                    <Button
                      primary
                      fullWidth={queries.isSmall}
                      {...primaryAction}
                    />
                  )}
                  {secondaryAction && (
                    <div className={styles.secondaryAction}>
                      <Button
                        secondary
                        fullWidth={queries.isSmall}
                        {...secondaryAction}
                      />
                    </div>
                  )}
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
      {text.footer && (
        <footer className={styles.footer}>
          <div className={styles.container}>
            <div className={styles.footerInner}>
              <div className={styles.poweredByContainer}>
                <Paragraph variant="extra-small" color="dark-reduced-opacity">
                  Powered by
                </Paragraph>
                <a
                  href="https://www.cultureamp.com"
                  className={styles.poweredByLogo}
                >
                  <img
                    src={assetUrl("brand/logo-horizontal-default.svg")}
                    alt="Culture Amp"
                  />
                </a>
              </div>
              <div className={styles.footerTextContainer}>
                <Paragraph variant="extra-small">{text.footer}</Paragraph>
              </div>
            </div>
          </div>
        </footer>
      )}
    </div>
  )
}

BrandMoment.displayName = "BrandMoment"
