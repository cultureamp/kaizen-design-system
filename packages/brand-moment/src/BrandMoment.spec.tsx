import * as React from "react"
import { render } from "@testing-library/react"
import { Box } from "@kaizen/component-library"
import { BrandMomentError } from "@kaizen/draft-illustration"
import { Paragraph } from "@kaizen/typography"
import { BrandMoment } from "./BrandMoment"

// eslint-disable-next-line ssr-friendly/no-dom-globals-in-module-scope
window.matchMedia = jest.fn().mockImplementation(() => ({
  matches: false,
  media: "",
  onchange: null,
  addListener: jest.fn(),
  addEventListener: jest.fn(),
  removeListener: jest.fn(),
  removeEventListener: jest.fn(),
}))

const mockPlay = jest.fn().mockResolvedValue(undefined)
const mockLoad = jest.fn()
const mockPause = jest.fn()

describe("<BrandMoment />", () => {
  beforeEach(() => {
    window.HTMLMediaElement.prototype.load = mockLoad
    window.HTMLMediaElement.prototype.play = mockPlay
    window.HTMLMediaElement.prototype.pause = mockPause
    Object.defineProperty(HTMLMediaElement.prototype, "muted", {
      set: jest.fn(),
    })
  })

  it("matches the snapshot", () => {
    const { container } = render(
      <BrandMoment
        body={
          <>
            <Box mb={1.75}>
              <Paragraph variant="intro-lede">
                This page cannot be found
              </Paragraph>
            </Box>
            <Paragraph color="dark-reduced-opacity" variant="small">
              Error code 404
            </Paragraph>
          </>
        }
        header={<></>}
        illustration={<BrandMomentError isAnimated loop />}
        mood="negative"
        primaryAction={{
          href: "/app/home",
          iconPosition: "end",
          label: "Go to Home",
        }}
        secondaryAction={{
          href: "mailto:info@cultureamp.com",
          label: "Contact support",
        }}
        text={{
          title: "Sorry but we can’t find the page you’re looking for.",
        }}
      />
    )
    expect(container).toMatchSnapshot()
  })
})
