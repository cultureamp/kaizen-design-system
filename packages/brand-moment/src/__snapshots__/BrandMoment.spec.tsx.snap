// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrandMoment /> matches the snapshot 1`] = `
<div>
  <div
    class="body negative"
  >
    <header
      class="header"
    />
    <main
      class="main"
    >
      <div
        class="container"
      >
        <div
          class="mainInner"
        >
          <div
            class="left"
          >
            <div
              class="leftInner"
            >
              <figure
                class="figure"
              >
                <video
                  aria-hidden="true"
                  autoplay=""
                  class="wrapper"
                  data-testid="kz-video-player"
                  loop=""
                  muted=""
                  playsinline=""
                  poster="https://d1e7r7b0lb8p4d.cloudfront.net/illustrations/heart/scene/brand-moments-error.png"
                  preload="metadata"
                  tabindex="-1"
                  width="100%"
                >
                  <source
                    src="https://d1e7r7b0lb8p4d.cloudfront.net/illustrations/heart/scene/brand-moments-error.mp4"
                    type="video/mp4"
                  />
                </video>
                <span
                  aria-live="polite"
                  class="container"
                >
                  <button
                    aria-label="Pause animation"
                    class="button default iconButton pausePlayButton"
                    type="button"
                  >
                    <span
                      class="content"
                    >
                      <span
                        class="iconWrapper"
                      >
                        <svg
                          aria-hidden="true"
                          class="icon"
                          focusable="false"
                          role="presentation"
                        >
                          <use
                            xlink:href="#undefined"
                          />
                        </svg>
                      </span>
                    </span>
                  </button>
                </span>
              </figure>
            </div>
          </div>
          <div
            class="right"
          >
            <div
              class="rightInner"
            >
              <div
                class="p-0 mb-1-point-5"
              >
                <h1
                  class="heading display-0 dark large"
                >
                  Sorry but we can’t find the page you’re looking for.
                </h1>
              </div>
              <div
                class="p-0 mb-1-point-5"
              >
                <div
                  class="p-0 mb-1-point-75"
                >
                  <p
                    class="paragraph intro-lede dark"
                  >
                    This page cannot be found
                  </p>
                </div>
                <p
                  class="paragraph small dark-reduced-opacity"
                >
                  Error code 404
                </p>
              </div>
              <div
                class="actions"
              >
                <span
                  aria-live="polite"
                  class="container"
                >
                  <a
                    class="button primary"
                    href="/app/home"
                    target="_self"
                  >
                    <span
                      class="content"
                    >
                      <span
                        class="label"
                      >
                        Go to Home
                      </span>
                    </span>
                  </a>
                </span>
                <div
                  class="secondaryAction"
                >
                  <span
                    aria-live="polite"
                    class="container"
                  >
                    <a
                      class="button secondary"
                      href="mailto:info@cultureamp.com"
                      target="_self"
                    >
                      <span
                        class="content"
                      >
                        <span
                          class="label"
                        >
                          Contact support
                        </span>
                      </span>
                    </a>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
  </div>
</div>
`;
